docs-src/docker.md

```diff
@@ -0,0 +1,89 @@
+# Docker for Kakunin tests
```

Comment:
Improve the description in `Docker for Kakunin tests` and `Dockerfile` sections. There are some grammar issues. @simonsipl